core/optaplanner-constraint-drl/src/main/java/org/optaplanner/constraint/drl/DrlScoreDirectorFactoryService.java

```diff
@@ -75,8 +82,55 @@ public Supplier<AbstractScoreDirectorFactory<Solution_, Score_>> buildScoreDirec
     }
 
     @Override
-    protected DrlScoreDirectorFactory<Solution_, Score_> createScoreDirectorFactory(ScoreDirectorFactoryConfig config,
-            SolutionDescriptor<Solution_> solutionDescriptor, KieBase kieBase) {
+    public ScoreDirectorType getSupportedScoreDirectorType() {
```

Comment:
Move this before `buildScoreDirectorFactory()` for structural consistency with other `ScoreDirectorFactoryService` implementations.